.eslintrc.cjs

```diff
@@ -432,6 +432,7 @@ const rules = {
   'jest/no-commented-out-tests': 1,
   'jest/no-conditional-expect': 1,
   'jest/no-conditional-in-test': 0,
+  'jest/no-confusing-set-timeout': 1,
```

Comment:
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-confusing-set-timeout.md

Reply:
Does vitest have an equivalent API?
https://vitest.dev/api/vi.html

Reply:
https://vitest.dev/guide/migration#timeout 
Should I disable it?

Reply:
Let's disable it, every enabled rule takes time to process